Misc/NEWS.d/next/Core_and_Builtins/2024-10-31-21-49-00.gh-issue-126072.o9k8Ns.rst

```diff
@@ -0,0 +1,2 @@
+Set ``ste_has_docstring`` for ``class`` and ``module`` in corresponding
```

Comment:
NEWS entries should focus on the user-visible effect, not on internal structures. In this case, we can say something about how the code objects for classes, modules, lambdas, and annotate functions no longer contain a None entry to signal the absence of a docstring.

Reply:
Actually it is.